.docker_utils/main_attach_dock.sh

```
@@ -3,3 +3,5 @@
 # Script to launch the main docker instance for the pblf110 car when the container was already created
 docker start forzaeth_racestack_ros2_humble
 docker attach forzaeth_racestack_ros2_humble
+-v /dev/input:/dev/input \
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)